src/App.tsx

```diff
@@ -1,17 +1,28 @@
-import React from 'react';
+import React, { useState, useMemo } from 'react';
 import './App.scss';
 import { MoviesList } from './components/MoviesList';
 import moviesFromServer from './api/movies.json';
 
 export const App: React.FC = () => {
+  const filteredMovies = [...moviesFromServer];
+  const [query, getQuery] = useState('');
+
+  const visibleMovies: Movie[] = useMemo(
+    () => filteredMovies.filter((movie) => {
+      return movie.title.toLowerCase().includes(query.toLowerCase())
+      || movie.description.toLowerCase().includes(query.toLowerCase());
```

Comment:
Trim the query to fix this problem
![image](https://user-images.githubusercontent.com/96409563/232307633-7a8e4826-b17f-46c9-8034-44873f348d11.png)